components/misc_metrics/menu_metrics.cc

```diff
@@ -37,7 +37,7 @@ const char* GetMenuGroupPrefKey(MenuGroup group) {
     case MenuGroup::kBrowserViews:
       return kBrowserViewsPrefKey;
   }
-  NOTREACHED_IN_MIGRATION();
+  return nullptr;
```

Comment:
why drop notreached?

Reply:
there is no default case in the switch, so any addition to the `MenuGroup` enum will force the dev to make a subsequent addition here